Apply color to TreeItem on focus and reduce indent in Tree

hbjORbj opened the pull request:
https://linear.app/intuita/issue/INT-797/apply-color-to-treeitem-on-focus-and-reduce-indent-in-tree

Before and After videos are inside the linear task above.

grzpab commented on intuita-webview/src/main/TreeView/index.tsx:
```diff
@@ -83,11 +94,13 @@ const TreeView = ({ node }: Props) => {
 				label={node.label ?? ''}
 				icon={icon}
-				indent={depth * 30}
+				indent={depth * 12}
```
what is the unit of it? `px`? `rem`?

hbjORbj replied:
in px. We do `<div style={{ minWidth: `${indent}px` }} />` when building `TreeItem`.

grzpab commented:
@hbjORbj actually I think that the initial "Recipe" node has too much whitespace on its left "bank", the rest of the icon placement is good. We could actually switch the folder icon with the one available in the vscode directly.

hbjORbj commented:
Addressed your comments. I add an icon for the ROOT kind as well. I think this icon makes sense because recipe does the fixes and the patches.

<img width="447" alt="Screenshot 2023-04-14 at 09 06 18" src="https://user-images.githubusercontent.com/32841130/231983671-97ad6cf2-0f5e-4ce6-b064-2d843b6a1a8e.png">

grzpab commented on intuita-webview/src/assets/wrench.svg:
who's the creator of that asset?

hbjORbj replied:
answered in slack
